### DISCUSSION TITLE
Custom Names For Objects and Items

### DISCUSSION THREAD

**Discussion by FarrunDragon** [FakeFishGames/Barotrauma#8696] [Ideas and suggestions]
I think a lot of depth/creativity could stem from being able to give custom names to objects (knives, masks, etc) and items (containers, machines, etc) in the sub editor and fabricator. I would love to hide special named weapons in loose vents and have my junction boxes be named according to their section, or doors named according to the room the open into, or infinite other uses.

**Comment by Misterzelix** [comment]
+1, custom names might have to be labels after the normal name to not confuse the AI
